src/twisted/test/test_defer.py

```diff
@@ -11,8 +11,17 @@
 import gc, traceback
 import re
 
+from twisted.python.compat import _PY3, _PY34PLUS
+
+if _PY34PLUS:
+    from asyncio import new_event_loop, Future, CancelledError
+    asyncSkip = None
+else:
+    asyncSkip = "asyncio not available before python 3.4"
+
```

Comment:
Hrm, that's fair.  I tend to think that it's better (and the intentions often clearer; though in this case the strings are pretty clear, so it's a thin argument here) to test for the think you need directly (import THING), and not indirectly (if python version is X then assume THING).